core/packages/test/scripts/start-ethereum.sh

```diff
@@ -12,7 +12,7 @@ start_chains()
 
 if [ -z "${from_start_services:-}" ]; then
     echo "start ethereum only!"
-    trap kill_ethereum SIGINT SIGTERM EXIT
+    trap kill_all SIGINT SIGTERM EXIT
```

Comment:
IMHO starting and killing chains separately could be helpful in dev stage when we only change some code/logic in parachain, then no need to restart ethereum and deploy contracts again